x/concentrated-liquidity/tick_bitmap.go

```diff
@@ -0,0 +1,117 @@
+package concentrated_liquidity
+
+import (
+	"fmt"
+	"math"
+	"math/bits"
+)
+
+// TickBitmap defines a bitmap used to represent price ticks. It contains a
+// mapping of 64-bit words, where each bit in a word corresponds to a unique
+// tick. A set bit, i.e. a bit set to 1, indicates liquidity for that tick.
+// Conversely, an unset bit means there is no liquidity for that tick. Note,
+// ticks are in the range [−887272, 887272].
+//
+// Ref: https://uniswapv3book.com/docs/introduction/uniswap-v3/#ticks
+// Ref: https://uniswapv3book.com/docs/milestone_2/tick-bitmap-index/#bitmap
+type TickBitmap struct {
+	bitmap map[int16]uint64
```

Comment:
How can we serialize and store this structure in the state?

Does it make sense to use a "prefix" map that we can store on disk?
e.g. we make a prefix of the format `bm< tick index >< key separator>< data >`